src/pages/Home.js

```diff
@@ -12,8 +12,30 @@ export default function Home(props) {
   function handleClick() {
     const token = getToken();
     addKeyValuePairToLocalStorage('token', token);
+    db.collection(token).add({
+      item_name: 'Your Item',
+      last_purchased: 'Soon, Later, Whenever',
+      purchase_frequency: null,
+    });
+    // .then((docRef) => {
+    //   console.log("New collection added with doc id: ", docRef.id);
+    // })
+    //
     props.setToken(token);
-    history.push('/list');
+    swal(
+      'List successfully created',
+      `Your new token is ${token}. Add your first item!`,
+    );
```

Comment:
Yay!  SweetAlert looking good! Is there any reason you didn't have the `success` property with the green check mark on this alert? Just curious!

Reply:
Total oversight! Thank you for the reminder 😁